recipe/meta.yaml

```diff
@@ -10,15 +10,15 @@ source:
   sha256: a370e7f25dbf05c9c151878c53556f27d0cbe7a4f909747db6e4b2d245f533cb
 
 build:
-  number: 0
+  number: 1
 
 outputs:
   - name: onednn
     script: build-onednn.sh  # [not win]
     script: bld-onednn.bat  # [win]
     version: {{ version }}
     build:
-      string: {{ dnnl_cpu_runtime }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
+      string: {{ dnnl_cpu_runtime }}{{ dnnl_experimental_ukernel }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
       run_exports:
         - {{ pin_subpackage("onednn", max_pin="x") }}
```

Comment:
This needs to change depending on experimental or not since the experimental ABI is unstable.

Reply:
You also need to add `track-features` to the experimental build to weigh it down in the solver. (Solver minimizes number of features in a given environment)

Reply:
@isuruf, could you verify whether I've done it correctly?